src/pages/Search/SearchSelectedNarrow.tsx

```diff
@@ -0,0 +1,74 @@
+import React, {useRef, useState} from 'react';
+import {Animated, View} from 'react-native';
+import type {DropdownOption} from '@components/ButtonWithDropdownMenu/types';
+import Icon from '@components/Icon';
+import MenuItem from '@components/MenuItem';
+import Modal from '@components/Modal';
+import PressableWithFeedback from '@components/Pressable/PressableWithFeedback';
+import type {SearchHeaderOptionValue} from '@components/Search/SearchPageHeader';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import variables from '@styles/variables';
+import * as Expensicons from '@src/components/Icon/Expensicons';
+import CONST from '@src/CONST';
+
+type SearchSelectedNarrowProps = {options: Array<DropdownOption<SearchHeaderOptionValue>>; itemsLength: number};
+
+function SearchSelectedNarrow({options, itemsLength}: SearchSelectedNarrowProps) {
+    const theme = useTheme();
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    const [isModalVisible, setIsModalVisible] = useState(false);
+    const buttonRef = useRef<HTMLDivElement>(null);
+
+    const openMenu = () => setIsModalVisible(true);
+    const closeMenu = () => setIsModalVisible(false);
+
+    return (
+        <View style={[styles.pb4]}>
+            <PressableWithFeedback
+                accessibilityLabel="selected"
+                ref={buttonRef}
+                style={[styles.tabSelectorButton, styles.ph5]}
+                onPress={openMenu}
+            >
+                {({hovered}) => (
+                    <Animated.View style={[styles.tabSelectorButton, styles.tabBackground(hovered, true, theme.border), styles.w100, styles.mh3]}>
+                        <View style={[styles.flexRow]}>
+                            <Text>{translate('workspace.common.selected', {selectedNumber: itemsLength})}</Text>
```

Comment:
`styles.bold` here?

```suggestion
                            <Text>{translate('workspace.common.selected', {selectedNumber: itemsLength})}</Text>
```

Reply:
Interesting, why does this use a custom component and not some kind of extension of our standard button components? Otherwise we run into issues just like this where we have inconsistent styles between buttons.

Reply:
Changed to Button

Reply:
Thanks!